internal/mode/static/telemetry/collector.go

```diff
@@ -6,12 +6,16 @@ import (
 	"fmt"
 
 	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/types"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 
 	"github.com/nginxinc/nginx-gateway-fabric/internal/mode/static/state/dataplane"
 	"github.com/nginxinc/nginx-gateway-fabric/internal/mode/static/state/graph"
 )
 
+// KubeSystem indicates the name of kube-system namespace
+const KubeSystem = "kube-system"
```

Comment:
nit: this shouldn't be exported if it's not used outside the telemetry package. In unit tests, you can just write `kube-system`